packages/google-maps/README.md

```diff
@@ -95,32 +92,40 @@ in `App_Resources/iOS/Info.plist`
 </plist>
 ```
 
+## Installation
+```cli
+npm install @nativescript/google-maps
+```
+
 ## Usage
+To show the map view, add the `<MapView>` element to your HTML. Then to manage the mapping features, get the reference to the [GoogleMap](#googlemap-object) instance from the `ready` event data.
 
 ### Core
 
-> **Important:** Ensure you've included xmlns:map="@nativescript/google-maps" on the Page element
-
-
 ```xml
-<map:MapView
-	lat="{{lat}}"
-	lng="{{lng}}"
-	zoom="{{zoom}}"
-	bearing="{{bearing}}"
-	tilt="{{tilt}}"
-	ready="{{onReady}}"
-	mapTap="{{onTap}}"
-	mapLongPress="{{onLongPress}}"
-	markerTap="{{onMarkerTap}}"
-/>
+<Page xmlns="http://schemas.nativescript.org/tns.xsd"
+  xmlns:map="@nativescript/google-maps">
+	<map:MapView
+		lat="{{lat}}"
+		lng="{{lng}}"
+		zoom="{{zoom}}"
+		bearing="{{bearing}}"
+		tilt="{{tilt}}"
+		ready="{{onReady}}"
+		mapTap="{{onTap}}"
+		mapLongPress="{{onLongPress}}"
+		markerTap="{{onMarkerTap}}"
+	/>
+</Page>
 ```
+To use the plugin in the different NativeScript flavors, modify the `main.ts` to import and then register it.
 
+> **Note** To handle the map features, see the [GoogleMap object](#googlemap-object) API.
```

Comment:
@herefishyfish  I think adding that note at the end of each flavor's example would be helpful.